### ISSUE TITLE
User Registration

### ISSUE THREAD

**Issue by profoundhub** [profoundhub/CoderSites#4]
Add ability for Users to register

**Comment by profoundhub** [comment]
Only Users who are signed in can edit, update, delete their own Campsites

**Comment by PJonestown** [comment]
Do you want to use a gem like monban or devise for this?

**Comment by profoundhub** [comment]
Yes, I will be adding that next, so people may register, and do CRUD
